src/cloud/faq-subscription-payments/faq-attach-vat-number.md

```diff
@@ -1,18 +1,26 @@
 ---
 zendesk:
   article_id: 26179753444381
-  name: I want my VAT number to be attached on the invoice
+  name: Adding VAT ID or Tax Information to Invoices
```

Comment:
```suggestion
  name: Adding VAT ID or tax information to invoices
```
We use [sentence-style capitalization](https://learn.microsoft.com/en-us/style-guide/capitalization), also in headings.